.github/workflows/check_code_format.yml

```diff
@@ -0,0 +1,25 @@
+#  This workflow will run the npm run check script to check if the formatting of the code is correct or not.
+name : check-code-formatting
```

Comment:
Can this name be in simple english? Since this is going to be displayed in the github UI

Like this:
![image](https://user-images.githubusercontent.com/56217868/114003032-0e1b4680-987b-11eb-836a-da4e318f81e9.png)

Reply:
Do I keep it as test itself? I thought something more verbal would be better, if not test please do suggest, so I can understand the convention for future

Reply:
Code format check with Prettier
Using this as it is easy and verbal